### PR TITLE
Add Pagination component

### PR DESCRIPTION
**Pull request by VitaliKozhurov**

### CODE REVIEW

File: src/styles/mixins.scss [hunk 1]
```diff
@@ -88,3 +88,18 @@
   border-radius: 50%;
   outline: none;
 }
+
+@mixin paginationItem {
+  display: flex;
```
Comment (DenisKulik) [inline src/styles/mixins.scss:93]: use flex mixin

File: src/styles/mixins.scss [hunk 2]
```diff
@@ -88,3 +88,18 @@
...
+  align-items: center;
+  justify-content: center;
+
+  box-sizing: border-box;
```
Comment (DenisKulik) [inline src/styles/mixins.scss:97]: border-box is set for all elements by default

**Review by DenisKulik — APPROVED** [review]

File: src/styles/mixins.scss (hunk 1 above)
Comment (VitaliKozhurov) [reply src/styles/mixins.scss]: Done!

File: src/styles/mixins.scss (hunk 2 above)
Comment (VitaliKozhurov) [reply src/styles/mixins.scss]: Done!

File: src/components/ui/pagination/Pagination.module.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+      cursor: pointer;
+
+      &.disabledController {
+        cursor: default;
+        color: var(--color-dark-100);
+      }
+
+      &:first-child {
+        margin-right: 12px;
```
Comment (vitalyabedik) [inline src/components/ui/pagination/Pagination.module.scss:48]: convert to rem, 52 row too

File: src/components/ui/pagination/createRangeArray.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+export const createRangeArray = (start: number, end: number): number[] => {
```
Comment (vitalyabedik) [inline src/components/ui/pagination/createRangeArray.ts:1]: need move to utils or helpers folder in common

File: src/components/ui/pagination/Pagination.module.scss (hunk 3 above)
Comment (VitaliKozhurov) [reply src/components/ui/pagination/Pagination.module.scss]: Done!

File: src/components/ui/pagination/createRangeArray.ts (hunk 4 above)
Comment (VitaliKozhurov) [reply src/components/ui/pagination/createRangeArray.ts]: Create new folder in common and move helper function

**Pull request merged**